core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -2806,6 +2806,100 @@ class KafkaApisTest {
     }
   }
 
+  @ParameterizedTest
```

Comment:
Were there previously no tests for this api? 😅

Reply:
Yeah... At least no unit tests here.